packages/compass-components/src/components/bson-value.tsx

```diff
@@ -2,8 +2,10 @@ import React, { useMemo } from 'react';
 import type { TypeCastMap, TypeCastTypes } from 'hadron-type-checker';
 import { Binary } from 'bson';
 import type { DBRef } from 'bson';
-import { css, cx, spacing } from '../';
+
 import { Icon, Link } from './leafygreen';
+import { spacing } from '@leafygreen-ui/tokens';
```

Comment:
This change and all the similar ones:
- remove circular imports from the root of compass-components

we encountered an issue with ESM modules in tests and this workaround fixed it for a bit. It didn't prove effective in the end, but since this change makes the way things are required inside compass-components consistent I didn't revert it.